server/utils/index.js

```diff
@@ -1,7 +0,0 @@
-const { KeyboardEncoder } = require('./keyboardencoder')
```

Comment:
why did you remove utility's? its handy to keep stuff organized and prevent clutter with import statements bc you can just say:
const {function1, function2} = require('./utils')